File: java/arr.java


public class arr{
public static void main(String args[])
{
  int[] an={0,1,2,3,4,5,6,7,8,9};
  int product;
  System.out.println("new array after operation\n");
for(int i=0;i<10;i++)
    {
     product= mul(an[i]);
     System.out.println(product);
    }
}

public static  int mul(int a)
{
   int d;
  d=a*3;
  return(d);
}
}

File: ruby/arr.rb
def arr(c)
s=Array.new(3)
for i in 0..2
 s[i]=Array.new(3)
  for j in 0..2
 s[i][j]=c[i][j]*3
end
 end
puts"array after operation\n"
for k in 0..2
  for l in 0..2
puts "#{s[k][l]}"
end 
 end
  end

  a=Array.new(3)
  
 
puts"enter a  array\n"
for row in 0..2
  a[row] = Array.new(3)	
  for col in 0..2
a[row][col]= gets.chomp.to_i;
end
 end
 arr(a)
